### ISSUE TITLE
Cannot load 0.3 release files with Python rdflib

### ISSUE THREAD

**Issue by karlcz** [dcppc/crosscut-metadata#16]
I see a related issue #3 which is already closed.  In that issue discussion I think the problem is even called out but seems not to be addressed.

When attempting to load either of the three datasets in the 0.3 release tarball, I encounter the same HTTP 404 error for the URL `https://datatagsuite.github.io/context/sdo/dataset_context.jsonld` which is not directly present in the JSON files but must be coming from one of the other layers of identifier resolution.

Unless I am confused here, I would expect this trivial code to work:

```
import rdflib
fname = 'KC7-crosscut-metadata-v0.3/data/datasets/GTEx_v7_rnaseq_public.json'
g = rdflib.Graph()
f = open(fname, 'r')
g.parse(file=f, format='json-ld')
```

I'm not an expert on rdflib, and not sure if there is a meaningful way to load these files into a triple store and inspect them in spite of this error. It actually surprises me that there isn't an obvious parse() parameter to parse the local file without attempting to retrieve any remote resources...

**Comment by carlkesselman** [comment]
This is actually a redirect from the path: https://w3id.org/dats/context/sdo/material_context.jsonld

which is specified in the @context declaration.  If you try to access this file just from the browser, no go.

**Comment by jonathancrabtree** [comment]
@agbeltran , is there any way to restore the previous context URIs for backwards compatibility, or are we going to mark this a "won't fix" and move on to v0.4 with the new URIs?

**Comment by jonathancrabtree** [comment]
The context URIs have been fixed in the conversion code and the corrected URIs should show up in the v0.4 release, which I will produce ASAP.

**Comment by jonathancrabtree** [comment]
This is fixed in v0.4. I don't think there's much point in trying to reverse engineer into v0.3 and earlier because:

* The schema.org-dependent context files in v0.3 and earlier didn't allow for traversing all the relationships in the JSON-LD data.
* The context files in https://github.com/datatagsuite/schema are not (AFAIK) versioned in tandem with this repo (or at all?), meaning the references would have to be to a specific GitHub commit. 

The latter issue may need to be addressed at some point.
